### ISSUE TITLE
icinga2 fails to start on fresh CentOS 7 install

### ISSUE THREAD

**Issue by hfinucane** [Icinga/chef-icinga2#311]
I see this in the journal:
```
    Sep 21 23:47:08 ip-172-29-1-231.us-west-2.compute.internal prepare-dirs[14350]: Try 'chmod --help' for more information.
    Sep 21 23:47:08 ip-172-29-1-231.us-west-2.compute.internal prepare-dirs[14350]: chmod: missing operand after ‘750’
    Sep 21 23:47:08 ip-172-29-1-231.us-west-2.compute.internal prepare-dirs[14350]: Try 'chown --help' for more information.
    Sep 21 23:47:08 ip-172-29-1-231.us-west-2.compute.internal prepare-dirs[14350]: chown: missing operand after ‘icinga:icinga’
    Sep 21 23:47:08 ip-172-29-1-231.us-west-2.compute.internal prepare-dirs[14350]: Try 'mkdir --help' for more information.
    Sep 21 23:47:08 ip-172-29-1-231.us-west-2.compute.internal prepare-dirs[14350]: mkdir: missing operand
```

I think it is because this cookbook is now out of sync with [this change](https://github.com/Icinga/icinga2/pull/5392/files), and [/etc/sysconfig/icinga2](https://github.com/Icinga/chef-icinga2/blob/master/templates/default/icinga2.service.config.erb) is missing `ICINGA2_CACHE_DIR`.

**Comment by vkhatri** [comment]
Thank you for reporting. Will look into it.

**Comment by scopenco** [comment]
My PR regarding this issue https://github.com/Icinga/chef-icinga2/pull/313

**Comment by vkhatri** [comment]
Closing, fixed by PR #313
